build_library/set_lsb_release

```diff
@@ -26,10 +26,9 @@ ROOT_FS_DIR="$FLAGS_root"
 [ -d "$ROOT_FS_DIR" ] || die "Root FS does not exist? ($ROOT_FS_DIR)"
 
 OS_NAME="Flatcar Container Linux by Kinvolk"
-OS_CODENAME="Oklo"
 OS_ID="flatcar"
 OS_ID_LIKE="coreos"
-OS_PRETTY_NAME="$OS_NAME $FLATCAR_VERSION (${OS_CODENAME})"
+OS_PRETTY_NAME="$OS_NAME $FLATCAR_VERSION"
```

Comment:
Maybe make it possible to have one but not mandatory
```
if [ "${OS_CODENAME}" = "" ]; then
  CODENAME_OPTION=""
else
  CODENAME_OPTION=" (${OS_CODENAME})"
fi
OS_PRETTY_NAME="$OS_NAME $FLATCAR_VERSION${CODENAME_OPTION}"
```

Reply:
after looking at this further, I'm thinking it'll be confusing to some for CODENAME to just disappear. And then again confusing for it to possibly re-appear?